server/app/controllers/fusor/api/v2/deployments_controller.rb

```diff
@@ -87,5 +89,42 @@ def find_deployment
     def customer_portal_credentials
       { :username => session[:portal_username], :password => session[:portal_password] }
     end
+
+    def save_deployment_attributes
+      path = ::Fusor.log_file_dir(@deployment.label, @deployment.id)
+      if !File.directory?(path)
+        Utils::Fusor::CommandUtils.run_command("mkdir -p #{path}")
+      end
+
+      dep_file = "#{path}/deployment.json"
+      if File.file?(dep_file)
+        Utils::Fusor::CommandUtils.run_command("rm -f #{dep_file}")
+      end
+
+      text = JSON.pretty_generate(@deployment.serializable_hash)
+      Fusor.log.info ("====== The following are the Deployment Attributes! ======")
+      Fusor.log.info ("#{text}")
+      begin
+        File.write(dep_file, text)
+      rescue SystemCallError
+        fail _("Failed to write Deployment Attributes to file!")
+      end
+
+      if @deployment.deploy_openstack
+        osp_file = "#{path}/openstack.json"
+        if File.file?(osp_file)
+          Utils::Fusor::CommandUtils.run_command("rm -f #{osp_file}")
+        end
```

Comment:
openstack deployments now have their own deployment object.. we need to save it.  I can combine the two, but thought a separate file would make CI, or QE's tasks easier.